connexion/decorators/security.py

```diff
@@ -222,6 +237,31 @@ def wrapper(request, required_scopes):
     return wrapper
 
 
+def verify_bearer(bearer_info_func):
+    def wrapper(request, required_scopes):
+        authorization = request.headers.get('Authorization')
+        if not authorization:
+            return None
+
+        try:
+            auth_type, token = authorization.split(None, 1)
+        except ValueError:
+            raise OAuthProblem(description='Invalid authorization header')
```

Comment:
Is `OAuthProblem` class appropriate here? The code seems more generic and not only about `OAuth`

Reply:
Previously connexion only supported oauth. It's more generic now. We could rename the Exception, but it would be backward incompatible if someone has set up a custom error_handler.

Reply:
You could also inherit from OAuthProblem in order to rename it and keep it backwards compatible.  (and leave a note to change this on the next major version bump).